swan-lake/development-tutorials/build-and-run/cicd.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: ballerina-cli-documentation-left-nav-pages-swanlake
+title: CI/CD for Ballerina Projects
+description: Learn how to create a CI/CD pipeline for a Ballerina project.
+keywords: ballerina, programming language, CICD, CI/CD, pipeline, github actions, gitlab, bitbucket
+permalink: /learn/build-and-run/ci-cd/
+active: cicd
+intro: This guide explains how to create a CI/CD pipeline for a Ballerina project.
+---
+
+
+Continuous Integration and Continuous Deployment (CI/CD) pipelines are essential for modern software development. They automate the process of building, testing, and deploying applications, ensuring faster delivery and higher quality. If you're working with Ballerina, a cloud-native programming language, setting up a CI/CD pipeline can streamline your development workflow. In this article, we’ll walk through how to create CI/CD pipelines for Ballerina projects using popular platforms like GitHub Actions, GitLab CI/CD, and Bitbucket Pipelines.
+
+
+### Why CI/CD for Ballerina?
+
+Ballerina is designed for building cloud-native applications, making it a great fit for modern development practices. A CI/CD pipeline for Ballerina can:
+
+- Automate testing to catch bugs early.
+- Build and package your application for deployment.
+- Push your Ballerina modules to Ballerina Central or container registries.
+- Ensure consistency across development, testing, and production environments.
+
+Let’s dive into setting up CI/CD pipelines for Ballerina projects.
+
```

Comment:
Extra new line?

Reply:
No. This is to separate the next section.